pkg/proto/runtime/v1/dapr_additional.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+// This file contains additional, hand-written methods added to the generated objects.
```

Comment:
Why is this file needed?
Is `SetComponentName` not automatically generated?

Reply:
It is not.

I need these methods so I can have an interface that matches those structs, and allows avoiding repetition of a lot of code.